apps/common/src/main/resources/db/migration/canton-network/postgres/stable/V043__scan_verdict_store.sql

```diff
@@ -0,0 +1,52 @@
+create table scan_verdict_store
+(
+    -- Primary key for the table
+    row_id                      bigint generated by default as identity primary key,
+    -- The time when this row was inserted, used for debugging and monitoring
+    ingested_at                 timestamptz not null default now(),
+    -- Migration identifier for domain migrations
+    migration_id                bigint not null,
+    -- Domain identifier
+    domain_id                   text not null,
+    -- When was this verdict sequenced
+    record_time                 bigint not null,
+    -- When was this verdict finalized
+    finalization_time           bigint not null,
+    -- Participant who submitted the verdict
+    submitting_participant_uid  text not null,
+    -- The result of the verdict
+    verdict_result              smallint not null,
+    -- Mediator group identifier
+    mediator_group              integer not null,
+    -- Update identifier for this verdict
+    update_id                   text not null,
+    -- Array of parties that submitted this verdict
+    submitting_parties          text[] not null,
+    -- Array of transaction root view IDs associated with this verdict
+    transaction_root_views      integer[] not null
+);
+
+-- Index for efficient querying by migration, and record time
+create index scan_verdict_mi_rt_di on scan_verdict_store (migration_id, record_time, domain_id);
```

Comment:
the comment does not fit the index (is `domain_id` needed?)

Reply:
Fixed, domain_id is not needed